eahub/config/settings.py

```diff
@@ -50,20 +50,21 @@ class DjangoEnv(Enum):
     "django_extensions",
     "rules.apps.AutodiscoverRulesConfig",
     "sorl.thumbnail",
-    "eahub.base.apps.BaseConfig",
-    "eahub.localgroups.apps.LocalGroupsConfig",
-    "eahub.profiles.apps.ProfilesConfig",
     "import_export",
     "rangefilter",
     "flags",
+    "rest_framework",
+    "eahub.base.apps.BaseConfig",
+    "eahub.localgroups.apps.LocalGroupsConfig",
+    "eahub.profiles.apps.ProfilesConfig",
 ]
 
 MIDDLEWARE = [
     "django.middleware.security.SecurityMiddleware",
     "whitenoise.middleware.WhiteNoiseMiddleware",
     "django.middleware.cache.UpdateCacheMiddleware",
     "django_referrer_policy.middleware.ReferrerPolicyMiddleware",
-    "django_feature_policy.FeaturePolicyMiddleware",
+    "django_feature_policy.PermissionsPolicyMiddleware",
```

Comment:
Why this change?

Reply:
"FeaturePolicyMiddleware" has been depracated 1-2 years ago, it's now called "permissions"